ras_party/views/account_view.py

```diff
@@ -164,14 +164,41 @@ def post_pending_survey_respondent():
     return make_response(jsonify(response), 201)
 
 
-@account_view.route("/respondents/<respondent_id>/password-verification-tokens", methods=["POST"])
-def post_password_verification_tokens(respondent_id):
+@account_view.route("/respondents/<respondent_id>/password-verification-token", methods=["GET"])
+def get_password_verification_token(respondent_id):
+    token = account_controller.get_respondent_password_token(respondent_id)
+    return make_response(jsonify({"token": token}), 200)
+
+
+@account_view.route("/respondents/<respondent_id>/password-verification-token", methods=["POST"])
```

Comment:
For the future, doing it this way is naughty, but I don't think it's a huge deal for this bit of work.
If you change the name of url in 2 services without allowing for backwards compatability (in this case, allowing for `-tokens` AND `-token`) then there will be a non-0 amount of time where the bit of functionality won't work.

For this one it's acceptable becuase it's for something that isn't constantly in use and will only stop working for a second.  But be more considerate in future as this'll bite you if it's a more core bit of functionality.

Reply:
For this one and the other one, can you add the old version as a route above it:

```python
@account_view.route("/respondents/<respondent_id>/password-verification-token", methods=["POST"])
@account_view.route("/respondents/<respondent_id>/password-verification-tokens", methods=["POST"])
def ....():
```
Put a comment somewhere around it that the `-tokens` route is deprecated
Also update the API documentation with the old route and also mention that it's deprecated.